.shellcheckrc

```diff
@@ -17,3 +17,6 @@ disable=SC2174
 
 # Disable warning of functions in test statements
 disable=SC2310
+
+# Disable warning of read without -r
```

Comment:
[SC2312](https://www.shellcheck.net/wiki/SC2312) checks for masking of return codes from chained commands, not `read`

[SC2162](https://www.shellcheck.net/wiki/SC2162) is the code that warns about `read` without `-r`